crates/symbolicator/src/services/objects/data_cache.rs

```diff
@@ -337,7 +373,7 @@ mod tests {
 
             // for each of the different symbol sources, we assert that:
             // * we get a negative cache result no matter how often we try
-            // * we hit the symbol source exactly once for the initial request
+            // * we hit the symbol source once for the initial request, then another 3 times to retry
```

Comment:
*another* 3 times, or 3 in total? I was under the impression that we try 3 times in total, so this might be wrong in a different place?

Reply:
It's another three times, I believe: 
https://github.com/getsentry/symbolicator/blob/1aa8a7a0aa9af4259a27a3a736a0cf874709bf6d/crates/symbolicator/src/utils/futures.rs#L290-L297

The check happens in the helper after the task is executed, so this retries 3 times in total.